mathlibtools/lib.py

```diff
@@ -634,6 +634,32 @@ def import_graph(self) -> 'ImportGraph':
         self._import_graph = G
         return G
 
+    def reduce_imports(self, file: str, sed: bool = False) -> Iterable[Any]:
```

Comment:
The return type annotation is too vague.

Reply:
I think it makes sense to split this into two functions, a non-sed version and a sed version that calls the non-sed version.

Reply:
That would probably be clearer indeed.

Reply:
Agreed